test/gtest/Makefile.am

```diff
@@ -107,6 +104,11 @@ gtest_SOURCES = \
 	ucs/test_twheel.cc \
 	ucs/test_frag_list.cc
 
+if ! ENABLE_SYMBOL_OVERRIDE
+gtest_SOURCES += \
+	ucm/malloc_hook.cc
```

Comment:
this only disables the test - i think we should disable most of UCM in this case, return `UCS_ERR_UNSUPPORTED` from `ucm_set_event_handler`, and the test should expect this behavior instead of being completely disabled.